hphp/CMakeLists.txt

```diff
@@ -28,6 +30,13 @@ if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/facebook")
                         ${CMAKE_MODULE_PATH})
 endif()
 
+find_package(PostgreSQL)
```

Comment:
I don't see any use of `find_package(PostgreSQL)` in any existing `CMakeLists.txt` or `*.cmake`.  I'll move it to HHVMExtensionConfig.cmake.